docs/providers/aws/guide/functions.md

```diff
@@ -25,6 +25,7 @@ service: myService
 provider:
   name: aws
   runtime: nodejs14.x
+  runtimeManagement: auto #optional, set how Lambda controls all functions runtime. AWS default is auto; this can either be 'auto' or 'onFunctionUpdate'. For 'manual', see example in hello function below (syntax for both is identical)
```

Comment:
Let's also add into this document section dedicated for runtime management (as e.g. we have one here for SnapStarrt: https://github.com/serverless/serverless/blob/776aa3b7fb563cfe0df5a2b09e528df1db599edf/docs/providers/aws/guide/functions.md#snapstart)